docs/docs-content/automation/palette-cli/commands/vmo.md

```diff
@@ -117,6 +128,125 @@ palette vmo import-ova --skip-image
 
 ## Migrate VM
 
+### Prerequisites
```

Comment:
@benradstone If these are the same prereqs then maybe we should use a partial? That way it's easier for us to ensure the information in synced 😄

Reply:
Will link to full guide as mentioned in the other comment.